sorc/ncep_post.fd/RQSTFLD.F

```diff
@@ -26,2728 +26,18 @@ module RQSTFLD_mod
       implicit none
 !
 !     increase MXFLD each time you add a new field
-
       INTEGER, PARAMETER :: MXFLD=1200
-
-#ifdef COMMCODE
-      INTEGER, PARAMETER :: MXLVL=70 
-#else
       INTEGER, PARAMETER :: MXLVL=500
-#endif
-      CHARACTER*20 AVBL(MXFLD),FIELD(MXFLD)
-      CHARACTER*50 AVBLGRB2(MXFLD)
+      
       CHARACTER*6 DATSET      
 !
-      LOGICAL RITEHD,RITE2
+      LOGICAL RITEHD
 !
-      integer :: KGTYPE,IOUTYP,SVALUE,NFLD,IGET(MXFLD),           &
-                 IQ(MXFLD),IS(MXFLD),ISMSTG(MXFLD),               &
-                 ISMFUL(MXFLD),ISMOUT(MXFLD),LVLS(MXLVL,MXFLD),   &
-                 IDENT(MXFLD),IFILV(MXFLD),IAVBLFLD(MXFLD),       &
-                 ID(25),IGDS(18)
```

Comment:
@hertneky May I ask the reason of removing "IGDS(18)"? It seems to me there are some setting igds for grib2 in grib2_module.f.

Reply:
@WenMeng-NOAA grib2_module.f does not use IGDS(18) from RQSTFLD.F. It initializes igds(5) on L514, separately.

Reply:
@hertneky You are right.